modules.d/01systemd-ask-password/module-setup.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Prerequisite check(s) for module.
+check() {
+
+    # If the binary(s) requirements are not fulfilled
+    # return 1 to not include the binary.
+    require_binaries systemd-ask-password || return 1
+    require_binaries systemd-tty-ask-password-agent || return 1
+
+    # If the module dependency requirements are not fulfilled
+    # return 1 to not include the required module(s).
+    if ! dracut_module_included "systemd"; then
+        derror "systemd-ask-password needs systemd in the initramfs."
+        return 1
+    fi
+
+    # Return 255 to only include the module, if another module requires it.
+    return 255
+
+}
+
+# Module dependency requirements.
+depends() {
+
+    # This module has external dependency on the systemd module.
+    echo systemd
+    # Return 0 to include the dependent systemd module in the initramfs.
+    return 0
+
+}
+
+# Install the required file(s) for the module in the initramfs.
+install() {
+
+    inst_multiple -o \
+
+        # Install the systemd type service unit for systemd-ask-password.
+        $systemdsystemunitdir/systemd-ask-password-console.path \
```

Comment:
This does not work in bash... line continuation is broken here due to newline and comment

Reply:
Fixed along with removing the module check(s)